composer.json

```
@@ -19,7 +19,7 @@
     "neos/neos-development-collection": "7.0.x-dev",
     "neos/flow-development-collection": "7.0.x-dev",
     "neos/demo": "@dev",
-    "neos/neos-ui": "@dev",
+    "neos/neos-ui": "^7.0@dev",
```

Do we maybe need to handle that package in the set-dependencies script?

---

Sounds like it… probably should have done that when we first decided to keep branches in sync for the UI package(s)…

---

```suggestion
    "neos/neos-ui": "7.0.x-dev",
```
then maybe to be in line with the others?

---

Well, it's not the same repo and I am not sure it's "version-synced". E.g. there is only UI 4.0.x for all of Neos 4.x. Then again, for v5 we have "all branches".